setup.py

```diff
@@ -0,0 +1,65 @@
+# This file is part of ts_dream_common.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import sys
+import setuptools  # type: ignore
+import pathlib
+from typing import List
+
+install_requires: List[str] = []
+tests_require: List[str] = [
+    "pytest",
+    "pytest-cov",
+    "pytest-flake8",
+    "pytest-black",
+    "pytest-mypy",
+]
+dev_requires: List[str] = install_requires + tests_require + ["documenteer[pipelines]"]
+scm_version_template = """# Generated by setuptools_scm
+__all__ = ["__version__"]
+
+__version__ = "{version}"
+"""
+tools_path = pathlib.PurePosixPath(setuptools.__path__[0])
```

Comment:
I don't think you need the following 3 lines... Consider removing them....

Reply:
Done and also two imports that become obsolete because of that.